vmdb/app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -16,6 +16,8 @@ class CloudManager < BaseManager
   has_many :cloud_object_store_objects,    :foreign_key => :ems_id, :dependent => :destroy
   has_many :key_pairs,                     :class_name  => "AuthPrivateKey", :as => :resource, :dependent => :destroy
 
+  alias all_cloud_networks cloud_networks
```

Comment:
Note: This alias was added so the `allowed_cloud_networks` method in miq_provision_cloud_workflow.rb can continue to call the same method on src_obj without having to treat the provider or cloud_tenant objects differently.